src/current/v24.1/create-function.md

```diff
@@ -194,6 +194,82 @@ SELECT last_rider();
 (1 row)
 ~~~
 
+### Create a function that uses `OUT` and `INOUT` parameters
+
+The following statement uses a combination of `OUT` and `INOUT` parameters to modify a provided value and output the result. An `OUT` parameter returns a value, while an `INOUT` parameter passes an input value and returns a value.
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE OR REPLACE FUNCTION double_triple(INOUT double INT, OUT triple INT) AS 
+  $$
+  BEGIN
+    double := double * 2;
+    triple := double * 3;
+  END;
+  $$ LANGUAGE PLpgSQL;
+~~~
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SELECT double_triple(1);
+~~~
+
+~~~
+  double_triple
+-----------------
+  (2,6)
+~~~
+
+Note that the `CREATE FUNCTION` statement does not need a `RETURN` statement because this is added implicitly for a function with `OUT` parameters:
```

Comment:
```suggestion
The `CREATE FUNCTION` statement does not need a `RETURN` statement because this is added implicitly for a function with `OUT` parameters:
```